server/controllers/users.js

```diff
@@ -1,6 +1,8 @@
 const jwt = require('jsonwebtoken');
 
 const User = require('../models').User;
+const Role = require('../models').Role;
+const UserRole = require('../models').UserRole;
 
```

Comment:
Isn't Babel configured for this project?

Reply:
What's Babel and what do we need it for?

Reply:
[Babel](https://babeljs.io) is a JavaScript transpiler, it converts modern JavaScript into the older versions that most browsers understand